app/styles/partials/event-card.scss

```diff
@@ -54,4 +54,4 @@
 
 .order.card {
   height: 100px !important;
```

Comment:
Try as min-height: 100px 
![Screenshot from 2021-02-04 15-31-18](https://user-images.githubusercontent.com/61330148/106876868-30b7a480-66fe-11eb-9ae9-07313223f770.png)

Reply:
Ohk got it. Let me try.

Reply:
But don't mind why did u change ui grid to ui menu and like mobile only row??

Reply:
Otherwise it's working fine. Plz let me know if I am wrong here.

Reply:
I did not change any style, its your deploy link that is opened.

Reply:
Alright. I have got this as screenshot below. Have a look
![c1](https://user-images.githubusercontent.com/11625672/106897613-2e167880-6719-11eb-861d-058c9aac50f8.JPG)

Reply:
Try reducing the width further

Reply:
Thanks, but it works like before I made changes while reducing the width further. U can checkout without deployment. BTW it is fine in desktop view and mobile view after your suggestions.

Reply:
**current -**
![Screenshot from 2021-02-04 19-44-34](https://user-images.githubusercontent.com/61330148/106904628-6d48c780-6721-11eb-87b7-ba02aac9d1a0.png)

**after your PR -** 
![Screenshot from 2021-02-04 19-44-10](https://user-images.githubusercontent.com/61330148/106904693-7fc30100-6721-11eb-8e70-71eabb58585c.png)

The issue is about the overflowing text which is still not resolved.

Reply:
Sorry for this, jst asking, this error happens when this page is in responsive mode but it's been fixed in both desktop and mobile view. If u want me to resolve that above issue, then there will be another error like height gets increased in order card causing extra wide space in the image area in desktop view.

Reply:
@MeghalBisht sorry for the misunderstanding. I got you now that it is to make further changes in responsive mode also.